template.env

```diff
@@ -1 +1,2 @@
-DISCORD_TOKEN=
\ No newline at end of file
+DISCORD_TOKEN=
+CONSUL_ADDRESS=172.17.0.1:8500
```

Comment:
for dev-env, this can just be `consul` because theyre in the same network namespace

Reply:
Oh right